custom_components/tryfi/pytryfi/fiPet.py

```diff
@@ -160,6 +190,93 @@ def updateAllDetails(self, sessionId: requests.Session):
         # TODO: Support weekly
         self._dailySleep, self._dailyNap = self._extractSleep(petJson['dailySleepStat'])
         self._monthlySleep, self._monthlyNap = self._extractSleep(petJson['monthlySleepStat'])
+        # Try to fetch behavior data for Series 3+ collars
+        try:
+            self.updateBehaviorStats(sessionId)
```

Comment:
Instead of making the service call, then seeing if it has any data, are we able to look at the FiDevice object and check to see if it's a series 3? That would avoid the extra call for series 1 and 2 collars.